new_platforms/vscode-extension/marketplace-homepage.md

```diff
@@ -0,0 +1,132 @@
+# Open Enclave extension for Visual Studio Code
+
+A [Visual Studio Code](https://code.visualstudio.com/) [extension](https://marketplace.visualstudio.com/VSCode) supporting [Open Enclave](https://github.com/Microsoft/openenclave), including development, debugging, emulators, and deployment!
+
+## Getting started
+
+You can install the Open Enclave extension into Visual Studio Code directly via the Visual Studio Gallery [TBD](TBD).
+
+Alternatively, you can run the extension from this repository by following these instructions:
+
+1. Clone this repository `git clone --recursive https://github.com/microsoft/openenclave`
+2. Navigate to `new_platforms\vscode-extension` in the cloned folder.
+3. Run npm to install the dependencies: `npm install` (see Requirements section for npm installation link)
+4. Start VSCode: `code .`
+5. Start the extension using F5
+
+## Features
+
+### Create a new Open Enclave solution.
+
+You can use the **Microsoft Open Enclave: New Open Enclave Solution** command (commands can be found using **CTRL-Shift-P**) 
+to create a new Open Enclave solution.  
+
+You will be prompted to:
+
+1. Select a folder for your solution.  
+1. On Linux, you will have the option to create a standalone project or an Azure IoT Edge container 
+    project.  Otherwise, you will only have the option to create an Azure IoT Edge container project.
+1. If you choose a container project, you will be prompted to provide your container repository.
+1. You will be prompted to provide a name for your host/enclave.
+
+A new solution will be created in the folder you've selected.  That solution will contain both the host
+and enclave as well as the required edl.  If you've chosen to build a container, the host will include
+some code that implements the required Azure IoT Hub communication.
+
+### Build your Open Enclave solution.
+
+There are build tasks for both standalone and Azure IoT Edge container projects.  The underlying system used 
+to build is CMake.  
+
+For a standalone project, there will be configure and build tasks for each target (arm32v7 and aarch64).  The 
+configure task will invoke CMake to create the required build files.  This is only required to be run once.  
+The build task will do the actual compiling and linking.
+
+1. **CTRL-Shift-P**
+1. Select `Tasks: Run Task`
+1. Select `Configure for QEMU (ARM | AARCH64)`
```

Comment:
Shouldn't "ARM" be "ARM32v7"?

Reply:
ok

Reply:
changed ARM to ARMV7-A and AARCH64 to AARCH64/ARMv8-A